app/feedstock/[id]/header.tsx

```diff
@@ -0,0 +1,74 @@
+import { BackButton } from '@/components/back-button'
+import { Feedstock } from '@/types/types'
+import { Column, Row } from '@carbonplan/components'
+import { alpha } from '@theme-ui/color'
+import { Box } from 'theme-ui'
+
+export const FeedstockHeader: React.FC<{ feedstock: Feedstock }> = ({
+  feedstock,
+}) => {
+  return (
+    <Box
+      sx={{
+        position: 'relative',
+        height: ['300px', '350px', '400px'],
+        width: '100%',
+        overflow: 'hidden',
+      }}
+    >
+      <Box
+        as='img'
+        // @ts-ignore
+        src={feedstock.thumbnail}
+        alt={`${feedstock.title} thumbnail`}
+        sx={{
+          position: 'absolute',
+          height: '100%',
+          width: '100%',
+          filter: 'grayscale(100%)',
+          objectFit: 'cover',
+        }}
+      />
+      <Box
+        sx={{
+          backgroundColor: alpha(feedstock.color, 0.5),
+          position: 'absolute',
+          height: '100%',
+          width: '100%',
+          opacity: 0.8,
+        }}
+      />
+      <Box
+        sx={{
+          position: 'absolute',
+          top: 0,
+          left: [3, 4, 5, 6],
+          right: 0,
+          bottom: 0,
+          display: 'flex',
+          alignItems: 'center',
```

Comment:
I think that we might want to avoid using `flex` here, or at least aligning using `center`. It'd be great if (a) the back button could be baseline aligned with the first line of the title and (b) that back button showed up in the same location for each feedstock (there's a lot of variability currently).

| ![CleanShot 2024-10-03 at 17 12 41@2x](https://github.com/user-attachments/assets/a68cf70d-acb5-49ab-b1eb-5989e0f9a175) | ![CleanShot 2024-10-03 at 17 12 35@2x](https://github.com/user-attachments/assets/c3f6b8be-2728-460e-adff-b0a616eed330) |
|-|-|

Reply:
Potentially related to the use of `flex`, the back button runs onto a second line on mobile:
![CleanShot 2024-10-03 at 17 18 05@2x](https://github.com/user-attachments/assets/299296e7-bc2a-4277-91c4-2e2bb496648e)